media-libs/svt-av1/svt-av1-0.8.4.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic cmake
+
+DESCRIPTION="Scalable Video Technology for AV1 (SVT-AV1 Encoder and Decoder)"
+HOMEPAGE="https://github.com/OpenVisualCloud/SVT-AV1"
+
+if [ ${PV} = "9999" ]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/OpenVisualCloud/SVT-AV1.git"
+else
+	SRC_URI="https://github.com/OpenVisualCloud/SVT-AV1/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/SVT-AV1-${PV}"
+fi
+
+LICENSE="BSD-2"
+IUSE="debug"
+SLOT="0"
+
+DEPEND="dev-lang/nasm"
+RDEPEND=""
+
+src_prepare() {
+	append-ldflags -Wl,-z,noexecstack
```

Comment:
is this really needed ?
if so, please add a comment citing a couple of files built with exec stacks

Reply:
```
 * QA Notice: The following files contain writable and executable sections
 *  Files with such sections will not work properly (or at all!) on some
 *  architectures/operating systems.  A bug should be filed at
 *  https://bugs.gentoo.org/ to make sure the issue is fixed.
 *  For more information, see:
 * 
 *    https://wiki.gentoo.org/wiki/Hardened/GNU_stack_quickstart
 * 
 *  Please include the following list of files in your report:
 *  Note: Bugs should be filed for the respective maintainers
 *  of the package in question and not hardened@gentoo.org.
 * RWX --- --- usr/lib64/libSvtAv1Dec.so.0.8.4
 * RWX --- --- usr/lib64/libSvtAv1Enc.so.0.8.4
```

Cheers

Reply:
hmmm so this is not being used when building ? it seems UNIX && !APPLE should be true for us :/